srcpkgs/python3-jupyter_nbconvert/template

```diff
@@ -1,28 +1,26 @@
 # Template file for 'python3-jupyter_nbconvert'
 pkgname=python3-jupyter_nbconvert
-version=6.0.7
-revision=2
+version=6.4.5
+revision=1
 wrksrc="nbconvert-${version}"
 build_style=python3-module
+# WebPDF export depends on unpackaged pyppeteer
+make_check_args="-m not(network) -k not(test_webpdf_without_chromium)"
 hostmakedepends="python3-setuptools"
 depends="python3-mistune python3-Jinja2 python3-jupyterlab_pygments
  python3-nbclient python3-jupyter_core python3-entrypoints python3-testpath
- python3-bleach python3-pandocfilters python3-defusedxml"
+ python3-bleach python3-pandocfilters python3-defusedxml
+ python3-BeautifulSoup4"
 checkdepends="python3-pytest python3-ipython_ipykernel
  python3-jupyter_ipywidgets python3-Pebble pandoc inkscape $depends"
 short_desc="Converting Jupyter notebooks"
 maintainer="Alessio Sergi <al3hex@gmail.com>"
 license="BSD-3-Clause"
 homepage="https://github.com/jupyter/nbconvert"
 distfiles="${PYPI_SITE}/n/nbconvert/nbconvert-${version}.tar.gz"
-checksum=cbbc13a86dfbd4d1b5dee106539de0795b4db156c894c2c5dc382062bbc29002
+checksum=21163a8e2073c07109ca8f398836e45efdba2aacea68d6f75a8a545fef070d4e
 conflicts="python-jupyter_nbconvert<=5.4.1_2"
```

Comment:
Can someone tell me if this`conflict` line should remain or not? Seems quite old stuff, we don't have a python2 package for this anymore.

Reply:
The issue is whether somebody still has `python-jupyter_nbconvert` installed; this defines an alternative that will conflict with the unversioned `/usr/bin/jupyter_nbconvert` shipped with the current package.